### ISSUE TITLE
Local read-only variables in bash scripts (`install-consul` and `run-consul`) are not declared correctly

### ISSUE THREAD

**Issue by opdecirkel** [hashicorp/terraform-google-consul#14]
In `install-consul` and `run-consul` there are many declarations in the form:
```
local readonly name=val
```
While syntax is accepted by bash, it does not make the variable readonly.
For local variable to be readonly, it should be declared as:
```
local -r name=val
```
Detailed explanation is provided in https://stackoverflow.com/questions/45409822/bash-local-and-readonly-variable/

**Comment by josh-padnick** [comment]
You know, we (Gruntwork) have used the `local readonly name=val` pattern in probably hundreds of bash scripts by this point, and it looks like you're exactly right:

```bash
# This script should return a non-zero exit code, but it doesn't.
function hello {
    local readonly name="$1"
    name="consul"

    echo "hello, $name"
}
```

We'll do as you suggest, not just in this repo, but across all repos we maintain and update to:

```bash
# This is the correct way to specify a readonly var in bash
function hello {
    local -r name="$1"
    name="consul"

    echo "hello, $name"
}
```

Thank you for pointing this out! I'll fix this in this repo momentarily.

**Comment by josh-padnick** [comment]
Correction. This is fixed in #12.
